src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderType.cs

```diff
@@ -14,22 +22,26 @@ public class Query : IRequest<Command>
         public Guid ProviderId { get; set; }
     }
 
-    public class Command : IRequest<Success>
+    public class Command : IRequest<OneOf<ModelWithErrors<Command>, Success>>
```

Comment:
I think the read and write model should be split up here - `ProviderTLevelDefinitions` shouldn't be in the write model, for me

Reply:
I was under the impression that it was only possible to bind a single model to an MVC view - happy to go through this offline.